schema/SessionAnswer.js

```diff
@@ -135,7 +136,7 @@ cube(`AggregateSessionMetrics`, {
       type: `avg`,
     },
     accuracy: {
-      sql: `100.0 * ${CUBE}.num_correct / NULLIF(${CUBE}.num_answered, 0)`,
+      sql: `100.0 * ${CUBE}.num_correct / NULLIF(${CUBE}.num_answered_evaluation, 0)`,
```

Comment:
`num_answered_evaluation` is not super clear to me. can we think of some other name?
This variable means the (number of questions answered) metric that is going to be used for evaluation, is that it?